Validation/RecoMuon/python/associators_cff.py

```diff
@@ -174,8 +174,8 @@
 #    acceptOneStubMatchings = cms.bool(False),
 #    rejectBadGlobal = cms.bool(True),
 ##############################################
-    PurityCut_track = 0.75,
-    PurityCut_muon = 0.75,
+    PurityCut_track = cms.double(0.75),
+    PurityCut_muon = cms.double(0.75),
```

Comment:
why is this needed?

Reply:
No reason in particular. The defaults were given in cms.double() form so I did the same